map.js

```diff
@@ -0,0 +1,115 @@
+$(document).ready(function() {
+  L.mapbox.accessToken = 'pk.eyJ1IjoiYWxla2diZyIsImEiOiJCUUs2UVlRIn0.O4MCWY-5DPT3p3eMidpcuQ';
+  
+  var Map = function () {
+    this.map = L.mapbox.map('map')
+      .setView([40, -100], 4)
+      .addLayer(L.mapbox.tileLayer('mapbox.streets'));
+    this.geocoder = L.mapbox.geocoder('mapbox.places');
+  };
+
+  Map.prototype = {
+    mapIt: function(startUp) {
+      var marker = L.marker(startUp.latlng, {
+          icon: L.mapbox.marker.icon({
+            'title': startUp.name,
+            'marker-color': '#f44',
+            'marker-symbol': 'star',
+            'marker-size': 'medium'
+          })
+        });
+      markers.addLayer(marker);
+    },
+    startUpData: function(id) {
+      $.ajax({
+        dataType: 'jsonp',
+        url: 'https://api.angel.co/1/tags/' + id + '/startups'
+      }).done(function(data) {
+        StartUp.prototype.initFromBundle(data.startups);
+      }).fail(function() {
+        console.log('error');
+      });
+    }, 
+    searchBox: function() {
```

Comment:
You might consider including a verb in the method names for this and the one above. Makes it more clear that something will change as a result.